src/arfsdao.ts

```diff
@@ -276,7 +276,7 @@ export class ArFSDAOAnonymous extends ArFSDAOType {
 		maxDepth: number,
 		includeRoot: boolean
 	): Promise<ArFSPublicFileOrFolderWithPaths[]> {
-		if (maxDepth < 0 || !Number.isInteger(maxDepth)) {
+		if (maxDepth < 0 || (!Number.isInteger(maxDepth) && maxDepth !== Number.POSITIVE_INFINITY)) {
```

Comment:
I used `Number.POSITIVE_INFINITY` as `maxDepth` when `--all` entities are requested. Because of that, I had to add this check to the condition. But now that we do have `ParametersHelper.getMaxDepth(): number` we could assert this in the input-reading step.

Do you guys think it's better to move this check to the `ParametersHelper.getMaxDepth()` so it can be removed here?

Reply:
I already changed this :)